src/distrs/gamma.jl

```diff
@@ -22,3 +25,58 @@ function gammalogpdf(k::T, θ::T, x::T) where {T<:Real}
     val = -loggamma(k) + xlogy(k - 1, xθ) - log(θ) - xθ
     return x < 0 ? oftype(val, -Inf) : val
 end
+
+gammacdf(k::T, θ::T, x::T) where {T<:Real} = first(gamma_inc(k, max(0, x)/θ))
+gammacdf(k::Real, θ::Real, x::Real)        = gammacdf(promote(float(k), θ, x)...)
```

Comment:
I think it would be better to promote first and convert to floating point numbers afterwards. Otherwise e.g. `gammacdf(1, 2f0, 5f0)` will return a `Float64`. So as below already for `gammalogcdf` maybe
```suggestion
gammacdf(k::Real, θ::Real, x::Real)        = gammacdf(map(float, promote(k, θ, x))...)
```

Reply:
But I wonder - is it actually necessary to convert to `float` here? Isn't this handled by `gamma_inc` in the line above?